deploy/mpijob-v1alpha2-crd.yaml

```diff
@@ -0,0 +1,5644 @@
+# --------------------------------------------------
+# - Configuration deployment YAML for MPI-Operator
+# - Include:
+#      CRD deployment 
+# --------------------------------------------------
+# The code is generated by crd-validation
```

Comment:
Are you using this? https://github.com/kubeflow/crd-validation
Does k8s provide other official solutions?

Reply:
yes, this validation is generated by [crd-validation](https://github.com/kubeflow/crd-validation).  This openAPIV3Schema should be the official solution to validate the CRD which include pod template.